testsuite/testcases/src/load_vs_perf_benchmark.rs

```diff
@@ -34,7 +34,9 @@ impl LoadVsPerfBenchmark {
         duration: Duration,
     ) -> Result<SingleRunStats> {
         let rng = SeedableRng::from_rng(ctx.core().rng())?;
-        let emit_job_request = ctx.emit_job.clone().mode(EmitJobMode::ConstTps { tps });
+        let emit_job_request = ctx.emit_job.clone().mode(EmitJobMode::MaxLoad {
```

Comment:
Do we want these changes as part of this PR?

Reply:
Undid it here. Let me do a follow up PR to get the best out of the consensus-only forge tests.